app/controllers/event_controller.rb

```diff
@@ -0,0 +1,52 @@
+class EventController < ApplicationController
+  before_action :session?
+  def index
+    @new_list = Event.where('events.event_date >= ?', DateTime.current.beginning_of_day)
+    @old_list = Event.where('events.event_date < ?', DateTime.current.beginning_of_day)
```

Comment:
You should use a [scope](https://guides.rubyonrails.org/active_record_querying.html#scopes) to handle upcoming and past events ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/event_date_requirement.png) from Odin)